### PR TITLE
reviewing the lesson objectives for the scheduler lesson

### PR DESCRIPTION
**Pull request by psteinb**

### CODE REVIEW

**Comment by psteinb** [comment]
@ChristinaLK @tkphd @jstaf 
I cleared away a lot of text and will start towards the end of this week implementing https://github.com/hpc-carpentry/hpc-intro/blob/gh-pages/lesson-outline.md#fundamentals-of-cluster-resources

File: _episodes/11-hpc-intro.md [hunk 1]
```diff
@@ -19,12 +19,6 @@ keypoints:
 
 ## Why Use These Computers?
 
-> ## What do you need?
->
-> Talk to your neighbor about your research. How does computing help you do your research? How could
-> more computing help you do more or better research?
-{: .challenge}
-
```
Comment (ChristinaLK) [inline _episodes/11-hpc-intro.md]: I think it's important to motivate the connection between people's work (which they really care about) and what more computers can do for them.  If not here, via this exercise, it should be somewhere else. Is it already?  If not, we need to make sure this idea is not lost.

File: _episodes/13-scheduler.md [hunk 2]
```diff
@@ -4,11 +4,13 @@ teaching: 45
 exercises: 30
 questions:
 - "What is a scheduler and why are they used?"
-- "How do we submit a job?"
+- "How do I launch a program to run on any one remote host in the cluster?"
+- "How do I capture the output of a program that is run on a remote host?"
```
Comment (ChristinaLK) [inline _episodes/13-scheduler.md]: Maybe use "computer" instead of "remote host"?

File: _episodes/13-scheduler.md [hunk 3]
```diff
@@ -4,11 +4,13 @@ teaching: 45
...
 objectives:
-- "Submit a job and have it complete successfully."
-- "Understand how to make resource requests."
-- "Submit an interactive job."
+- "Run a simple Hello World style program on the cluster."
+- "Submit a simple Hello World style script to the cluster."
+- "Use the batch system command line tools to monitor the execution of your job."
+- "Inspect the output and error files of your jobs."
```
Comment (ChristinaLK) [inline _episodes/13-scheduler.md]: I like these better. :)

File: _episodes/11-hpc-intro.md (hunk 1 above)
Comment (psteinb) [reply _episodes/11-hpc-intro.md]: ok, I get the point. Did that with ce3e91c

File: _episodes/13-scheduler.md (hunk 2 above)
Comment (psteinb) [reply _episodes/13-scheduler.md]: I'd personally would favor `node`, see 2b7de7a. what do you think?

File: _episodes/13-scheduler.md (hunk 3 above)
Comment (psteinb) [reply _episodes/13-scheduler.md]: thanks.

File: _episodes/13-scheduler.md (hunk 2 above)
Comment (ChristinaLK) [reply _episodes/13-scheduler.md]: That sounds good to me too.

**Comment by psteinb** [comment]
@ChristinaLK I started adding the introduction to the story with 1dc4400 and a paragraph on explaining laptop-vs-server-vs-cluster-vs-cloud (according to [lesson-outline.md](https://github.com/hpc-carpentry/hpc-intro/blob/gh-pages/lesson-outline.md#fundamentals-of-cluster-resources). Please provide feedback, so that I am going in the right direction. Fingers crossed.

**Comment by psteinb** [comment]
I added a first try for explaining the difference between using a laptop versus using a server versus using a cluster/supercomputer with 2ae5c00.
I'd love to have a review on this before I go for a merge.

File: _episodes/11-hpc-intro.md [hunk 4]
```diff
@@ -40,83 +34,61 @@ they started:
...
+
+Today, people coding or analysing data typically work with laptops.
+ 
+{% include figure.html url="" max-width="20%" file="/fig/200px-laptop-openclipartorg-aoguerrero.svg" alt="A standard laptop"
+caption="" %}
+
+While the following discussion may appear silly at first glance, let's dissect what resources programs running on a laptop require:
+- the keyboard and touchpad is used to provide flags and options
+- the display is shows progress and results
+- the laptops internal computing power and memory is used to perform calculations
```
Comment (ChristinaLK) [inline _episodes/11-hpc-intro.md]: I'd introduce the components of cpus / memory / disk here, even.
Maybe.

File: _episodes/11-hpc-intro.md [hunk 5]
```diff
@@ -40,83 +34,61 @@ they started:
...
+- offen GUI style human interfaces are employed
+
+### When tasks take too long
+
+When the task to solve become more heavy on computations, the operations are typically out-sourced from the local hardware in the laptop. Take for example the task to find the directions for your next business trip. The capabilities of your laptop are typically not enough to calculate that route spontaneously. So you use website, which in turn runs on a server that is almost exclusively no in the same room than you are.
```
Comment (ChristinaLK) [inline _episodes/11-hpc-intro.md]: hardware = jargon that needs to be defined.  What do you mean?

File: _episodes/11-hpc-intro.md [hunk 6]
```diff
@@ -40,83 +34,61 @@ they started:
...
+
+{% include figure.html url="" max-width="20%" file="/fig/servers-openclipartorg-ericlemerdy.svg" alt="A rack half full with servers"
+caption="" %}
+
+Note here, that a server is mostly a noisy computer mounted into a rack cabinet which in turn resides in a data center. The internet made it possible that these data centers do not require to be nearby your laptop. 
+What people call **the cloud** is mostly a web-service where you can rent such servers by providing your credit card details and by clicking together the specs of this remote resource.
+
+The server itself has no direct display or input methods attached to it. But most importantly, it has much more storage, memory and compute capacity than your laptop will ever have. In any case, you need a local device (laptop, workstation, mobile phone or tablet) to interact with this remote machine, people typically call 'a server'. 
+
```
Comment (ChristinaLK) [inline _episodes/11-hpc-intro.md]: I like this progression from: your computer -> computers that you rent that are elsewhere -> groups of computers

File: _episodes/11-hpc-intro.md [hunk 7]
```diff
@@ -40,83 +34,61 @@ they started:
...
+### When one server is not enough
+
+If the computational task or analysis to complete is daunting for a single server, larger agglomerations of servers are used. These go by the name of clusters or super computers.
+
+{% include figure.html url="" max-width="20%" file="/fig/serverrack-openclipartorg-psteinb-basedon-ericlemerdy.svg" alt="A rack full with servers"
+caption="" %}
+
+The methodology of providing the input data, communicating options and flags as well as retrieving the results is quite opposite to using a plain laptop. Moreover, using a GUI style interface is often discarded in favor of using the command line. This imposes a double paradigm shift for prospect users:
```
Comment (ChristinaLK) [inline _episodes/11-hpc-intro.md]: Again, maybe a shift away from jargon to more plain English, but the idea is good.

File: _episodes/11-hpc-intro.md [hunk 8]
```diff
@@ -40,83 +34,61 @@ they started:
...
+
+### A standard Laptop for standard tasks
+
+Today, people coding or analysing data typically work with laptops.
+ 
+{% include figure.html url="" max-width="20%" file="/fig/200px-laptop-openclipartorg-aoguerrero.svg" alt="A standard laptop"
+caption="" %}
+
+While the following discussion may appear silly at first glance, let's dissect what resources programs running on a laptop require:
+- the keyboard and touchpad is used to provide flags and options
```
[the remark this answers is not in the document]
Comment (psteinb) [reply _episodes/11-hpc-intro.md]: done

File: _episodes/11-hpc-intro.md (hunk 5 above)
Comment (psteinb) [reply _episodes/11-hpc-intro.md]: removed that bit to avoid confusion.

File: _episodes/11-hpc-intro.md (hunk 7 above)
Comment (psteinb) [reply _episodes/11-hpc-intro.md]: I need help with this.

**Comment by psteinb** [comment]
I will merge now, so that we can move on to the other lessons. PRs are highly welcome improving this part. I'd especially encourage artistically inclined collaborators, to improve the figures.

**Pull request merged**